src/tests/electron/tests/android-setup-locate-adb.test.ts

```diff
@@ -31,18 +31,20 @@ describe('Android setup - locate adb', () => {
         }
     });
 
-    it('respects user-provided adb location', async () => {
+    it('respects user-provided adb location, detect-adb passes a11y check', async () => {
         const [closeId, nextId] = [leftFooterButtonAutomationId, rightFooterButtonAutomationId];
         expect(await dialog.isEnabled(getAutomationIdSelector(closeId))).toBe(true);
         expect(await dialog.isEnabled(getAutomationIdSelector(nextId))).toBe(false);
 
-        await setupMockAdb(commonAdbConfigs['single-device']);
+        await setupMockAdb(commonAdbConfigs['slow-single-device']);
```

Comment:
Somehow it feels odd to end the test in the middle of a call to mock-adb. What if we allowed mock-adb to finish, like this?
```suggestion
        await setupMockAdb(simulateNoDevicesConnected(commonAdbConfigs['slow-single-device']));
```